# release rate and velocity ground truth value from box

#header
Header header

float64 releaseRate # release rate out of box
float64 windRate    # velocity of the wind out of box